Scheduler/observing_blocks_maintel/AOS/BLOCK-T345.json

```diff
@@ -0,0 +1,78 @@
+{
+    "name": "AOSSurveyMode",
+    "program": "BLOCK-345",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "maintel/track_target.py",
+            "standard": true,
+            "parameters": {
+                "target_name": "$name",
+                "slew_icrs": {
+                    "ra": "$ra",
+                    "dec": "$dec"
+                },
+                "rot_value": "$rot",
+                "rot_type": "PhysicalSky",
+                "az_wrap_strategy": "NOUNWRAP"
+            }
+        },
+        {
+            "name": "maintel/close_loop_comcam.py",
```

Comment:
The unit test is failing because the filter property is marked as required in the JSON schema for `BaseClosedLoop`.